src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -5569,4 +5503,41 @@ export class StudyViewPageStore {
         onError: error => Promise.resolve([]),
         default: [],
     });
+
+    public getChartTitle(chartType: ChartTypeEnum, title?: string) {
```

Comment:
i think this needs a good comment to explain why the molecularProfileSampleCountSet result impacts the title